.github/workflows/release_version.yaml

```diff
@@ -95,6 +95,7 @@ jobs:
           branch: version/${{ github.event.inputs.version }}
           delete-branch: true
           token: ${{ secrets.GITHUB_TOKEN }}
+          team-reviewers: team-helios
```

Comment:
Do we really need this? We have a `CODEOWNERS` file for that.

Reply:
Thats a good Q: This is more of a release which is automated and we are using a action `https://github.com/peter-evans/create-pull-request` which does not support code owners

Reply:
Looks like this action just open a PR right? afaik, Codeowners add team members to any PR openend in the repository, regardless of who opened it (e.g., look at PR opened by renovate, which is a bot as well. We are all added without any extra configuration required thanks to the codeowners file).

Reply:
This ticket got created the 1st of June, while our CODEOWNERS file got added the 21st of June, looks like the ticket is invalid for me, maybe @DennisBerthold-leanix can comment on that, since he created the ticket.

Reply:
Here you are: https://github.com/leanix/leanix-k8s-connector/pull/300

Reply:
@QuentinVaucherLeanIX Thanks. That saved a round trip.